test/CMakeLists.txt

```diff
@@ -43,12 +43,12 @@ set_tests_properties(CheckBuildManual PROPERTIES FAIL_REGULAR_EXPRESSION "ERROR"
 ## ERROR Tests
 ##
 
-if(NOT BOINC)
+if(NOT CR)
```

Comment:
Do you know why these don't work?

Reply:
Yes. But C/R version is very messy, and there is no point whatsoever to run this for C/R.

Reply:
#860